scripts/OverlaySurvey.py

```diff
@@ -232,6 +237,39 @@ def run_survey(args):
         json.dump(merged_results, outfile)
 
 
+def simplify(args):
+    simplified_graph = []
+    graph = nx.read_graphml(args.graphmlInput)
+    bidirectional_adj = {}
+    for v in graph.nodes():
+        if v not in bidirectional_adj:
+            bidirectional_adj[v] = set()
+        for w in graph.adj[v]:
+            if w not in bidirectional_adj:
+                bidirectional_adj[w] = set()
+            bidirectional_adj[v].add(w)
+            bidirectional_adj[w].add(v)
+    for node, attr in graph.nodes(data=True):
+        new_attr = {"publicKey": node}
+        appears_on_sb = False
+        for key in attr:
+            if key.startswith("sb_"):
+                new_key = key[3:]
+                appears_on_sb = True
+            else:
+                new_key = key
+            try:
+                new_attr[new_key] = json.loads(attr[key])
+            except Exception:
```

Comment:
you should catch a specific exception type that `json.loads` throws, `JSONDecodeError` from what I see

Reply:
Fixed!